packages/build/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "27.0.1",
   "description": "Netlify build module",
   "type": "module",
-  "exports": "./src/core/main.js",
+  "exports": {
```

Comment:
To minimise the changes to the main path, can we keep `exports` and export `runCoreStep` from `src/core/main.js`?

Something like:

```js
export { runCoreStep } from '../steps/run_core_step.js'
```